docs/docsite/rst/dev_guide/core_branches_and_tags.rst

```diff
@@ -0,0 +1,54 @@
+.. _core_branches_and_tags:
```

Comment:
If you choose NOT to move all of this into the development_process.rst, then please add `:orphan:` to the top of this page to prevent docs build warnings.